features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/metric/slowframes/DefaultSlowFramesListenerTest.kt

```diff
@@ -432,6 +448,162 @@ internal class DefaultSlowFramesListenerTest {
         }
     }
 
+    @Test
+    fun `M proxy to metricDispatcher W onViewCreated`(
+        @StringForgery viewId: String,
+        @LongForgery(min = 0) startedTimestampNs: Long
+    ) {
+        // When
+        testedListener.onViewCreated(viewId, startedTimestampNs)
+
+        // Then
+        verify(mockMetricDispatcher).onViewCreated(viewId)
+    }
+
+    @Test
+    fun `M incSlowFrame W onFrame { isJank = true }`(
+        forge: Forge
+    ) {
+        // Given
+        testedListener.onViewCreated(viewId, forge.aLong(min = 0))
+
+        // When
+        testedListener.onFrame(forge.aFrameData(isJank = true))
+
+        // Then
+        verify(mockMetricDispatcher).incSlowFrame(viewId)
+        verify(mockMetricDispatcher, times(0)).incIgnoredFrame(viewId)
```

Comment:
(also other occurrences)
```suggestion
        verify(mockMetricDispatcher, never()).incIgnoredFrame(viewId)
```